# Remote Management Request
temoto_core/RMPRequest trr

# The name of the robot to load 
string robot_name

---

# Remote Management Response
temoto_core/RMPResponse trr
